adapters/repos/db/replication.go

```diff
@@ -367,17 +369,14 @@ func (db *DB) DigestObjects(ctx context.Context,
 	class, shardName string, ids []strfmt.UUID,
 ) (result []replica.RepairResponse, err error) {
 	index := db.GetIndex(schema.ClassName(class))
-	if index == nil {
```

Comment:
Is removing the check for nil index intended?

Reply:
nope wasn't intended, got catch